CONTRIBUTING.md

```diff
@@ -72,6 +75,56 @@ If you do not find the suggestion, please open a new issue and clearly document
 
 Please provide your enhancement suggestions as an issue in the repository:
 
+### Dev Environments
+
+#### Manual Setup
+
+Instructions for setting up a local development environment in a virtual environment:
+
+```bash
+# Checkout the repository
+git clone https://github.com/cytomining/pycytominer.git
+cd pycytominer
+
+# Install pyenv if not already installed
+curl https://pyenv.run | bash
```

Comment:
I think it is worth providing some guidance on manual setup of development environments within this PR. If someone follows the recommended approach for their manual dev environment setup, it will put them on the same page in terms of auto-formatting of code contributions as people using devcontainers. I did however, add some caveats and clarifying language around this particular approach that makes it clear that the specific commands are intended for a linux dev environment and shifting the responsibility to virtual environment management back to the user.